e2e/rbd_helper.go

```diff
@@ -40,6 +40,18 @@ var nbdResizeSupport = []util.KernelVersion{
 	}, // standard 5.3+ versions
 }
 
+// nolint:gomnd // numbers specify Kernel versions.
+var fastDiffSupport = []util.KernelVersion{
+	{
+		Version:      5,
+		PatchLevel:   3,
+		SubLevel:     0,
+		ExtraVersion: 0,
+		Distribution: "",
+		Backport:     false,
+	}, // standard 5.3+ versions
```

Comment:
Do we know backport Centos kernel version?

Reply:
no idea about this one.

Reply:
This runs inside minikube for the CI in this repo, the CentOS kernel is not very relevant (unless someone runs it on an other environment).